zproject/settings.py

```diff
@@ -15,6 +15,9 @@
 # See https://zulip.readthedocs.io/en/latest/subsystems/settings.html for more information
 #
 ########################################################################
+import django_stubs_ext
+
+django_stubs_ext.monkeypatch()
```

Comment:
Can you add a comment documenting what this does?

https://pypi.org/project/django-stubs-ext/ explains it allows runtime monkey-patching... but it'd be helpful to have the context on why Zulip needs that behavior.

Reply:
As far as I know, we only really need this for `models.Lookup`. Added comments on this.